lib/logging/include/hse/logging/logging.h

```diff
@@ -19,6 +21,12 @@
 #define LOG_DEFAULT LOG_DEBUG
 #endif
 
+#ifdef HSE_REL_SRC_DIR
+#define LOG_FILE(x) (x + sizeof(HSE_REL_SRC_DIR) - 1)
+#else
+#define LOG_FILE(x) x
+#endif
```

Comment:
Use a descriptive arg name with a single leading underscore.

Reply:
It's not clear to me that calling this macro with anything other than __FILE__ makes sense...

Reply:
The only thing I don't like here is that I have repeated the macro 3 times, logging, ev, perfc. Any thoughts on where a good play to put this macro is? `compiler.h`

Reply:
Yeah, I saw that :-)   Yeah, it's definitely a buildenv/compiler thing, so that makes sense...